examples/inspection/plot_model_inspection_cross_validation.py

```diff
@@ -0,0 +1,235 @@
+"""
+=====================================================
+Inspect and analyze estimator within cross-validation
+=====================================================
+
+Evaluating a predictive model involves cross-validation. This example:
+
+* recalls the question that a cross-validation framework allows to answer to;
+* highlights the way to inspect the internals of a model when using the
+  cross-validation framework.
+"""
+
+print(__doc__)
+
+# %%
+# Dataset
+# -------
+#
+# We will use the california housing dataset where the goals is to predict the
+# average house value in a neighborhood. From the start, we will split our data
+# into two sets: a set that we will use to make all our experiments and a set
+# that we will leave out for further confirmation.
+
+# %%
+from sklearn.datasets import fetch_california_housing
+from sklearn.model_selection import train_test_split
+
+X, y = fetch_california_housing(as_frame=True, return_X_y=True)
+y -= y.mean()
+
+X_train, X_test, y_train, y_test = train_test_split(
+    X, y, test_size=0.1, random_state=0
+)
+
+# %%
+# Predictive model
+# ----------------
+#
+# In this example, we will use a linear model which should be a good baseline:
+# a ridge model. A ridge model enforce a L2 penalty on the coefficients. Thus,
+# the penalty parameter `alpha` has to be tuned. More importantly, this
+# parameter needs to be tune for our specific problem: a fine tuning on another
+# dataset does not ensure an optimal parameter value for the current dataset.
+#
+# Here, we use the class :class:`~sklearn.linear_model.RidgeCV` that allows to
+# tune `alpha` by cross-validation.
+#
+# Besides, we add a preprocessing stage to standardize the data such that the
+# optimization problem encountered by the ridge regressor is well-posed.
+
+# %%
+import numpy as np
+from sklearn.linear_model import RidgeCV
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+
+alphas = np.linspace(0.1, 100, num=30)
```

Comment:
I find it weird to use a linearly uniform grid to explore alpha. I think a log uniform grid would make more sense. Can you please try again with something like the following:

```suggestion
alphas = np.logspace(-3, 3, num=30)  # 10^-3 to 10^3
```

If there are some optimal alphas on the boundaries of that grid, I would even extend it further to [-6, 6] in log space.